boards/nucleo_f429zi/openocd.cfg

```diff
@@ -12,4 +12,3 @@ hla_vid_pid 0x0483 0x374b
 set WORKAREASIZE 0x10000
 
 source [find target/stm32f4x.cfg]
-
```

Comment:
revert

Reply:
Revert to what? As far as I remember, I never edited this file

Reply:
If github is showing a change, then the file changed. Looks like a line was removed.